src/api.ts

```diff
@@ -85,6 +86,85 @@ function getCollectKeyFromCommand(command: string): string|undefined {
   }
 }
 
+// function validateRequest(
+//   command: 'account_info',
+//   params: AccountInfoRequest
+// ): void
+// function validateRequest(
+//   command: 'gateway_balances',
+//   params: GatewayBalancesRequest
+// ): void
+
+/**
+ * Validate the request.
+ *
+ * Throws if `params` are invalid for `command`.
+ */
+function validateRequest(command: string, params: any): void {
```

Comment:
This isn't necessary in the new `request()` world, it should be the responsibility of the API to tell the user what was wrong with their request. Otherwise we end up duplicating and having to maintain validation logic in two separate places.

(This did make sense in the old world & still makes sense for the legacy methods, where `getSettings()`/`getAccountInfo()` were all special and being defined by the SDK).